dist/ramda.js

```diff
@@ -4440,6 +4440,43 @@
         }, [], fns);
     };
 
+    // The algorithm used to handle cyclic structures is heavily
+    // inspired by underscore.js' implementation of isEqual
+    var _eqDeep = function _eqDeep(a, b, stackA, stackB) {
+        var idx = stackA.length;
+        while (--idx >= 0) {
```

Comment:
Though it's overly tricky in my view, Ramda style is to use `while (idx--)` here, I believe. The postfix `--` makes it possible to rely on type coercion, avoiding the comparison operator.